src/earl-loader.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+const globby = require('globby')
+
+/**
+ * Load jsonld/earl report(s)
+ * - if there are multiple reports, this function concatanates them into one report
+ *
+ * @method earlLoader
+ * @param {String|String[]} pathOrGlob file path to jsonld/earl report
+ * @returns {Object[]}
+ */
+const earlLoader = async pathOrGlob => {
+	if (!pathOrGlob) {
+		throw new Error(`argument is missing`)
+	}
+
+	const reports = globby.sync(pathOrGlob).map(reportPath => {
+		const fileContent = fs.readFileSync(reportPath, { encoding: 'utf-8' })
+		return JSON.parse(fileContent)
+	})
+
+	return reports
+}
+
+module.exports = earlLoader
```

Comment:
This function does nothing with EARL. Calling it an earlLoader is a misnomer. It's a generic data loader. This either needs to do some checking or processing of the data, and filter out things that aren't EARL, or this needs to have a name. Either way the name doesn't describe what this thing does, so that needs to be sorted.

Reply:
I agree. I am going to call this `reportLoader`